srcpkgs/firefox-ublock-origin/template

```diff
@@ -0,0 +1,34 @@
+# Template file for 'firefox-ublock-origin'
+pkgname=firefox-ublock-origin
+version=1.62.0
+revision=1
+hostmakedepends="python3 zip"
+depends="firefox"
+short_desc="Effecient blocker for Firefox"
+maintainer="Emil Tomczyk <emru@emru.xyz>"
+license="GPL-3.0-or-later"
+homepage="https://ublockorigin.com/"
+_uassets_master="5722991a9ad362eaf4f2bb6ff64f08cccbafa2ef"
+_uassets_ghpages="2dde847869577e87fb9e0e095ac25323eba26560"
+distfiles="https://github.com/gorhill/uBlock/archive/refs/tags/${version}.tar.gz
+https://github.com/uBlockOrigin/uAssets/archive/${_uassets_master}.tar.gz
+https://github.com/uBlockOrigin/uAssets/archive/${_uassets_ghpages}.tar.gz"
+checksum="1f9d69f968ddec2ccfd3b104a3e0cda80403efd1a63356924406cf5296cde67b
+ a04e029394796104f60c346625707e1740069df8aea7e35af6d679b7b6febccb
+ afb5a554eefe4f4f8ecb8217d29518df71c76f1e85a10ffbd788c61e400c736d"
+
+post_extract() {
+	mkdir -p uBlock-${version}/dist/build/uAssets
+	mv uAssets-${_uassets_master} uBlock-${version}/dist/build/uAssets/main
+	mv uAssets-${_uassets_ghpages} uBlock-${version}/dist/build/uAssets/prod
+}
+
+do_build() {
+	cd uBlock-${version}
+	make firefox
+}
```

Comment:
This is unnecessary, wheter you adopt `vsrcextract` or not. Without `vsrcextract`, you probably need to set `build_wrksrc`. In either case, just set `make_build_target="firefox"`.